app/cdap/components/hydrator/components/TopPanel/NameAndDescription.tsx

```diff
@@ -14,32 +14,37 @@
  * the License.
  */
 
-import { Button, Tooltip, withStyles } from '@material-ui/core';
+import { Button, TextField, Tooltip, withStyles } from '@material-ui/core';
+import IconSVG from 'components/shared/IconSVG';
 import React, { useState } from 'react';
 import styled from 'styled-components';
 
 interface INameAndDescriptionProps {
   metadataExpanded?: boolean;
+  globals?: any;
```

Comment:
not sure if it's doable here, but try to avoid using `any` where possible.

Reply:
updated